content/html/concepts/tags/terms/embed/embed.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: '<embed>'
+Description: 'The <embed> tag allows you to "Embed" a video, image, page and more.'
```

Comment:
```suggestion
Description: 'Inserts a video, image, page, and more.'
```

For all the Terms entries in Docs, the `Description` field in the metadata should start with the verb.

Also added an Oxford comma.